Gems/LmbrCentral/Code/Source/Shape/EditorReferenceShapeComponent.h

```diff
@@ -8,24 +8,24 @@
 
 #pragma once
 
-#include <Components/ReferenceShapeComponent.h>
-#include <Vegetation/Editor/EditorVegetationComponentBase.h>
+#include<LmbrCentral/Component/EditorWrappedComponentBase.h>
+#include <Shape/ReferenceShapeComponent.h>
 
-namespace Vegetation
+namespace LmbrCentral
 {
     class EditorReferenceShapeComponent
-        : public EditorVegetationComponentBase<ReferenceShapeComponent, ReferenceShapeConfig>
+        : public EditorWrappedComponentBase<ReferenceShapeComponent, ReferenceShapeConfig>
     {
     public:
-        using BaseClassType = EditorVegetationComponentBase<ReferenceShapeComponent, ReferenceShapeConfig>;
+        using BaseClassType = EditorWrappedComponentBase<ReferenceShapeComponent, ReferenceShapeConfig>;
         AZ_EDITOR_COMPONENT(EditorReferenceShapeComponent, EditorReferenceShapeComponentTypeId, BaseClassType);
         static void Reflect(AZ::ReflectContext* context);
 
-        static constexpr const char* const s_categoryName = "Vegetation";
-        static constexpr const char* const s_componentName = "Vegetation Reference Shape";
+        static constexpr const char* const s_categoryName = "Shape";
+        static constexpr const char* const s_componentName = "Reference Shape";
```

Comment:
I just talked with UX, and they suggested the display name should be "Shape Reference" instead of "Reference Shape", which I'd agree makes more sense.  We can leave the file names the same, but just change the display name here.

Reply:
Changed